write-fonts/src/lib.rs

```diff
@@ -22,14 +22,23 @@
 //!
 //! ## Loading and modifying fonts
 //!
-//! Although `write-fonts` does not contain any parsing logic, it does offer the
-//! [`FromTableRef`] and [`ToOwnedTable`] traits (similar to `std`'s `From` & `Into`)
-//! for converting from a `read_fonts` type to its `write-fonts` equivalent.
-//! This means that you can read an existing font table into `write-fonts`;
-//! under the hood we will use `read-fonts` to parse the font, and then convert
-//! that to the `write-fonts` version. In general you do not need to think about
-//! this conversion yourself; tables implement the [`FontRead`] trait from
-//! `read-fonts`, which handles the reading + conversion logic for you.
+//! When modifying a font, you will typically start by reading the font data
+//! using `read-fonts`. When you come to write the font out, however, you will
+//! quickly discover that tables generated by `read-fonts` have different
+//! types to those required by `write-fonts`. This is because `read-fonts` types
+//! borrow their data from an underlying backing store, but `write-fonts` types
+//! need to own their data. In order to modify a table parsed by `read-fonts`,
+//! you will need to convert it to a `write-fonts` type. This can be done with
+//! the `FromTableRef` and `ToOwnedTable` traits: bring these traits into scope
+//! and call `.to_owned_table()` or `.to_owned_obj()` on the table or subtable
+//! you want to modify. For example:
+//!
+//! ```no_run
+//! let mut head: Head = font.head().expect("missing 'head' table").to_owned_table();
```

Comment:
```suggestion
//! ```no_run
//! use write_fonts::from_obj::ToOwnedTable;
//! let mut head: Head = font.head().expect("missing 'head' table").to_owned_table();
```